Description: Fixes "trap cleanup EXIT" in growpart
 When called from the busybox of the initramfs, the below code:
  trap cleanup EXIT
 fails. This patch replaces "EXIT" by zero (eg: 0). In Ubuntu, it seems that
 the busybox shell has such option, so it's not failing.
 .
 Thanks to Scott Moser for his help. Without him I probably wouldn't have find
 where to patch.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2013-06-11

--- cloud-utils-0.24.orig/bin/growpart
+++ cloud-utils-0.24/bin/growpart
@@ -109,7 +109,7 @@ done
 [ "${part#*[!0-9]}" = "${part}" ] || fail "partition-number must be a number"
 
 mktemp_d && TEMP_D="${_RET}" || fail "failed to make temp dir"
-trap cleanup EXIT
+trap cleanup 0
 
 change_out=${TEMP_D}/change.out
 change_err=${TEMP_D}/change.err
